Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/ContestsExportController.cs

```diff
@@ -103,7 +103,19 @@ public ActionResult Results(int id, bool compete)
                 headerRow.CreateCell(columnNumber++).SetCellValue(problem.Name);
             }
 
-            headerRow.CreateCell(columnNumber++).SetCellValue("Total");
+            var totalPointsCellTitle = "Total";
+
+            if (contest.IsOnline)
+            {
+                var maxPointsForOnlineContest = contest.Problems
+                    .GroupBy(p => p.GroupNumber)
+                    .Select(gr => gr.First())
+                    .Sum(p => p.MaximumPoints);
+
+                totalPointsCellTitle = $"{totalPointsCellTitle} (Max: {maxPointsForOnlineContest})";
```

Comment:
Can you check with a trainer, just in case it turns out they use the exported document to import it into the suls and changes in the name of columns could potentially break the import.